.cascade-code/Chapman.edu/_cascade/formats/modular/widgets/Funnels_1up

```diff
@@ -0,0 +1,68 @@
+##
```

Comment:
In this file, I moved the funnel link markup into a private macro for readability. Other than that, I just changed the indentation, and on line 24 I changed the heading from `h3` to `h2`